platforms/corda-ent/charts/float/templates/configmap.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.nodeName }}-conf
+  namespace: {{ .Values.metadata.namespace }}
+  labels:
+    app.kubernetes.io/name: {{ .Values.nodeName }}-conf
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+data:
+  firewall.conf: |+
+{{ tpl (.Files.Get "files/firewall.conf") . | indent 4 }}
```

Comment:
Add newline

Reply:
Fixed in [9862bdb](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/893/commits/9862bdbebd85ac9508aaec96a1807981550cf571)